app/build.gradle

```diff
@@ -66,7 +66,7 @@ dependencies {
     implementation project(':client')
     implementation 'androidx.appcompat:appcompat:1.5.1'
     implementation 'androidx.core:core-splashscreen:1.0.0'
-    implementation 'com.google.android.material:material:1.4.0'
```

Comment:
> On the login page I had to change the hint in the URL edittext from the example URL to a title like "Gotify URL".
That's because the hint moves to the top of the EditText if you input something. That would be misleading if there would be that example URL.

> On the settings page I had to overwrite the list preference method to use the Material You dialogs

I'm okay with that.